src/slic3r/GUI/GLCanvas3D.cpp

```diff
@@ -7403,37 +7403,17 @@ void GLCanvas3D::_check_and_update_toolbar_icon_scale()
         return;
     }
 
-    float scale = wxGetApp().toolbar_icon_scale();
-    Size cnv_size = get_canvas_size();
-
-    //BBS: GUI refactor: GLToolbar
-    int size_i = int(GLToolbar::Default_Icons_Size * scale);
-    // force even size
-    if (size_i % 2 != 0)
-        size_i -= 1;
-    float size   = size_i;
-
-    // Set current size for all top toolbars. It will be used for next calculations
-#if ENABLE_RETINA_GL
-    const float sc = m_retina_helper->get_scale_factor() * scale;
-    //BBS: GUI refactor: GLToolbar
-    m_main_toolbar.set_scale(sc);
-    m_assemble_view_toolbar.set_scale(sc);
-    m_separator_toolbar.set_scale(sc);
-    collapse_toolbar.set_scale(sc / 2.0);
-    size *= m_retina_helper->get_scale_factor();
-
+    /* Soolbar scale is a constant value which depends on display resolution and OS. */
```

Comment:
Could you add a prefex `Orca: ` for the comments?
So that it's will make my future work of merging upstreams easier